### PR TITLE
1226 create showcase app

### PR DESCRIPTION
**Pull request by Jordanafoster6**
This PR adds a "showcase" angular application that allows users to quicky spin up the UI of the ArcGIS plugin within mage server

### CODE REVIEW

File: plugins/arcgis/web-app/projects/main/src/lib/EventsResult.ts
Comment (newmanw) [inline plugins/arcgis/web-app/projects/main/src/lib/EventsResult.ts:1]: mockArcGISEventResult should live in the showcase application not the library

File: plugins/arcgis/web-app/projects/main/src/lib/arc-event/arc-event.component.ts
Comment (newmanw) [inline plugins/arcgis/web-app/projects/main/src/lib/arc-event/arc-event.component.ts:1]: I don't think we need to create a separate observable here, we are already listening to changes on `@Input config`

File: plugins/arcgis/web-app/projects/main/src/lib/arc.service.mock.ts
Comment (newmanw) [inline plugins/arcgis/web-app/projects/main/src/lib/arc.service.mock.ts:1]: This should live in the showcase application

File: plugins/arcgis/web-app/projects/showcase/src/app/app.component.ts
Comment (newmanw) [inline plugins/arcgis/web-app/projects/showcase/src/app/app.component.ts:1]: title property is not used

File: plugins/arcgis/web-app/projects/showcase/src/app/app.component.scss
Comment (newmanw) [inline plugins/arcgis/web-app/projects/showcase/src/app/app.component.scss:1]: Don't need to import styles from lib, these are exported as part of the component.  You will be missing a few global styles that are current part of the mage application.  Don't worry about these as they will be removed in the near future as we upgrade the admin application from AngularJS to Angular.

**Review by newmanw — CHANGES_REQUESTED** [review]
Should not need to modify or add anything in the library when creating the showcase application.

File: plugins/arcgis/web-app/projects/showcase/src/app/app.component.html
Comment (newmanw) [inline plugins/arcgis/web-app/projects/showcase/src/app/app.component.html:1]: arc-event and arc-layer are embedded components in arc-admin.  You can remove these from this template.

**Review by newmanw — CHANGES_REQUESTED** [review]
Few more things to resolve

**Comment by Jordanafoster6** [comment]
> Should not need to modify or add anything in the library when creating the showcase application.

The only thing i added to the plugin code was the typescript interface that the main service and the mock service (now in the showcase) implement. Everything else should be removed at this point.

**Review by newmanw — APPROVED** [review]

**Comment by gardnershawn379-create** [comment]
https://acrobat.adobe.com/id/urn:aaid:sc:EU:9028c566-e9a3-439c-be4d-25c347cf24b9

**Pull request merged**